screenpy/actions/either.py

```diff
@@ -53,10 +53,11 @@ def perform_as(self, the_actor: Actor) -> None:
         with the_narrator.mic_cable_kinked():
             try:
                 the_actor.will(*self.try_performables)
-                return
             except self.ignore_exceptions:
                 if not settings.UNABRIDGED_NARRATION:
                     the_narrator.clear_backup()
+            else:
```

Comment:
Why add this?

Reply:
It was the recommendation of one of these... Tryceratops, i think?

The intention is to make it clear what bits you expect to raise an exception and what bits you want to run if the exception was not raised.